packages/extensions/upward-security-headers/package.json

```
@@ -1,25 +1,25 @@
 {
-  "name": "@magento/upward-security-headers",
-  "version": "0.0.1",
-  "publishConfig": {
-    "access": "public"
-  },
-  "description": "Add security headers to UPWARD",
-  "main": "intercept.js",
-  "scripts": {
-    "clean": " "
-  },
-  "repository": "github:magento/pwa-studio",
-  "author": "Magento Commerce",
-  "license": "(OSL-3.0 OR AFL-3.0)",
-  "peerDependencies": {
-    "@magento/pwa-buildpack": "^5.1.1",
-    "@magento/venia-ui": "^3.0.0",
-    "webpack": "~4.38.0"
-  },
-  "pwa-studio": {
-    "targets": {
-      "intercept": "./intercept"
+    "name": "@magento/upward-security-headers",
+    "version": "0.0.1",
+    "publishConfig": {
+        "access": "public"
+    },
+    "description": "Add security headers to UPWARD",
+    "main": "intercept.js",
+    "scripts": {
+        "clean": " "
+    },
+    "repository": "github:magento/pwa-studio",
+    "author": "Magento Commerce",
+    "license": "(OSL-3.0 OR AFL-3.0)",
+    "peerDependencies": {
+        "@magento/pwa-buildpack": "^6.0.0",
+        "@magento/venia-ui": "^4.0.0",
```

Also weird, your update didn't use the `save-prefix` in `.yarnrc`; we're still on the tilde train.

```suggestion
        "@magento/pwa-buildpack": "~6.0.0",
        "@magento/venia-ui": "~4.0.0",
```

---

ALLLLLLLLrighty then. https://github.com/magento/pwa-studio/pull/2605/commits/5ee885fcfe5ab2c4c7173bb268f9b4ea065e3ec0.